server/cp/ingest/parser/globenewswire.py

```diff
@@ -0,0 +1,88 @@
+
+import logging
+import lxml.etree as etree
+import lxml.html as lxml_html
+
+from superdesk.text_utils import get_word_count
+from superdesk.io.feed_parsers.newsml_2_0 import NewsMLTwoFeedParser
+
+
+DESCRIPTION = 'Press Release'
+KEYWORD_ROLE = 'MWKeyRole:Ticker'
+BODY_FOOTER = {
+    'en': 'NEWS RELEASE TRANSMITTED BY Globe Newswire',
+    'fr': 'COMMUNIQUE DE PRESSE TRANSMIS PAR Globe Newswire',
+}
+NS = {
+    'iptc': 'http://iptc.org/std/nar/2006-10-01/',
+    'xhtml': 'http://www.w3.org/1999/xhtml',
+}
+
+logger = logging.getLogger(__name__)
+
+
+class GlobeNewswireParser(NewsMLTwoFeedParser):
+
+    NAME = 'globenewswire'
+    label = 'Globe Newswire'
+
+    def parse_content_meta(self, tree, item):
+        meta = super().parse_content_meta(tree, item)
+
+        item['language'] = item['language'].split('-')[0]
+
+        item['slugline'] = 'GNW-{lang}-{time}--{symbols}'.format(
+            lang=item['language'],
+            time=meta.find(self.qname('contentCreated')).text[17:19],
+            symbols='-'.join(self._get_stock_symbols(tree)),
+        )
+
+        keywords = meta.findall(self.qname('keyword'))
+        item['keywords'] = [k.text for k in keywords if k.text and k.get('role') == KEYWORD_ROLE]
+
+        item['description_text'] = DESCRIPTION
```

Comment:
Does it make sense to translate a description?

Reply:
I think it does, but don't have such specs